Resources/Prototypes/Catalog/VendingMachines/Inventories/dinnerware.yml

```diff
@@ -8,7 +8,9 @@
     FoodPlateSmall: 10
     FoodPlateTin: 5
     FoodKebabSkewer: 5
-    DrinkGlass: 10
+    DrinkGlass: 5
+    Beaker: 5
+    LargeBeaker: 5
```

Comment:
large containers are currently obtainable in high quantities from the booze-o-mat

shakers are 100u, as well as a large variety of the various bottles of booze like vodka and rum bottles (notably not beer bottles). the reason for this pr is so that you dont need to get a bottle from the booze-o-mat (failrp) to have a large container.

large beakers are also obtainable for 2 sheets of glass at science and start in the kitchen at roundstart on marathon. if that isn't intentional thats cool but also out of scope for the pr